diff --git a/node_modules/next/dist/build/collect-build-traces.js b/node_modules/next/dist/build/collect-build-traces.js
index 7aedde7..725e83e 100644
--- a/node_modules/next/dist/build/collect-build-traces.js
+++ b/node_modules/next/dist/build/collect-build-traces.js
@@ -343,7 +343,7 @@ async function collectBuildTraces({ dir, config, distDir, pageInfos, staticPages
                     try {
                         return await _promises.default.readlink(p);
                     } catch (e) {
-                        if ((0, _iserror.default)(e) && (e.code === "EINVAL" || e.code === "ENOENT" || e.code === "UNKNOWN")) {
+                        if ((0, _iserror.default)(e) && (e.code === "EINVAL" || e.code === "ENOENT" || e.code === "UNKNOWN" || e.code === "EISDIR")) {
                             return null;
                         }
                         throw e;
diff --git a/node_modules/next/dist/build/webpack/plugins/next-trace-entrypoints-plugin.js b/node_modules/next/dist/build/webpack/plugins/next-trace-entrypoints-plugin.js
index 043eb2e..0114039 100644
--- a/node_modules/next/dist/build/webpack/plugins/next-trace-entrypoints-plugin.js
+++ b/node_modules/next/dist/build/webpack/plugins/next-trace-entrypoints-plugin.js
@@ -397,7 +397,7 @@ class TraceEntryPointsPlugin {
                         });
                     });
                 } catch (e) {
-                    if ((0, _iserror.default)(e) && (e.code === "EINVAL" || e.code === "ENOENT" || e.code === "UNKNOWN")) {
+                    if ((0, _iserror.default)(e) && (e.code === "EINVAL" || e.code === "ENOENT" || e.code === "UNKNOWN" || e.code === "EISDIR")) {
                         return null;
                     }
                     throw e;
